src/ensembl/src/shared/eye-toggle-icon/EyeToggleIcon.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import eyeOnIcon from './eye-on.svg';
+import eyeOffIcon from './eye-off.svg';
```

Comment:
We have been using 2 svg's (as images) everywhere, but I think this should be done by

- first, importing the svg as a React component
- secondly, by applying various classes to the svg to change its fill property.

_(see the CloseButton in the Storybook as an example)._

The only drawback to this approach is that the svg will become part of the corresponding javascript file and make it a bit larger.

Reply:
This is something I intend to look at for this sprint, which ends only by end of next week. Most of these icons will get converted into React components using `svgr` and make use of classes to change to the different icon states.